insights/parsers/ironic_conf.py

```diff
@@ -0,0 +1,48 @@
+
+"""
+IronicConf - file ``/etc/ironic/ironic.conf``
+=============================================
+This class provides parsing for the file ``/etc/ironic/ironic.conf``.
+See the ``IniConfigFile`` class for more usage information.
+"""
+
+
+from .. import IniConfigFile, parser, add_filter
+from insights.specs import Specs
+
+add_filter(Specs.ironic_conf, ["["])
+
+
+@parser(Specs.ironic_conf)
+class IronicConf(IniConfigFile): 
+    """
+    Ironic configuration parser class, based on the ``IniConfigFile`` class.
+    Sample input data is in the format::
+            [DEFAULT]
```

Comment:
You have a document rendering error do to a missing blank line before `[DEFAULT]`.  Please test the rendering of your documentation before resubmitting following the instructions in the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#documentation-guidelines).